src/main/java/com/kodilla/ecommercee/controller/GroupNotFoundException.java

```diff
@@ -0,0 +1,5 @@
+package com.kodilla.ecommercee.controller;
+
+public class GroupNotFoundException extends Exception {
+   // System.out.printlm("Group does not exist");
```

Comment:
Zostawianie zakomentowanego kodu w kodzie produkcyjnym nie jest dobrą praktyką.
Warto by za to stworzyć w tym wyjątku konstruktor bezparametrowy, który wywoływałby konstruktor z klasy nadrzędnej z odpowiednim komunikatem, czyli np. `super("Product group not found.");`. Wtedy przy wywaleniu tego wyjątku, w response do klienta pójdzie właśnie zdefiniowana wiadomość, zamiast ogólnej `Internal server error`, która nie mówi niczego na temat przyczyny powstania wyjątku.